build/govuk_template/views/layouts/govuk_template.html

```diff
@@ -71,25 +71,29 @@
 
     {{{ content }}}
 
-    <footer class="group js-footer" id="footer" role="contentinfo">
+    <footer class="govuk-footer js-footer" role="contentinfo">
```

Comment:
We don't need to make these changes to the `build` file - only to the file in the main `layouts/` folder. We're not using this file anymore, and it could be over-written by someone running `grunt govuk_template`.